sklearn/datasets/tests/test_samples_generator.py

```diff
@@ -344,20 +344,20 @@ def test_make_hastie_10_2():
     assert np.unique(y).shape == (2,), "Unexpected number of classes"
 
 
-def test_make_regression():
+def test_make_regression(global_random_seed):
     X, y, c = make_regression(
-        n_samples=100,
+        n_samples=200,
```

Comment:
I think it's okay to increase the size of the dataset here since we're testing the variance of the generated data, and so we need sufficient data points for it to be stable enough.